src/components/ProductList/ProductList.js

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import axios from 'axios';
+import OfferBox from 'components/OfferBox/OfferBox';
+
+const baseURL = 'https://61a7c977387ab200171d2ec9.mockapi.io/list';
+
+const ProductList = () => {
+
+  const [products, setProducts] = useState([])
+
+  const getProducts = () => {
```

Comment:
jest dobrze, ale brakuje tutaj useEffect, teraz będziesz miała błąd, że za często wołasz api :) A potrzebujesz zrobić tego calla tylko 1 raz. Tutaj masz prosty przykład (ten 1 wystarczy), jakbyś miała z tym problem to daj znać :) 

https://dev.to/darkmavis1980/fetching-data-with-react-hooks-and-axios-114h